py3status/modules/weather_owm.py

```diff
@@ -817,6 +830,9 @@ def _format(self, current_wthr, fcsts, city, country):
         return self.py3.safe_format(self.format, today)
 
     def weather_owm(self):
+        # Prepare rate limit cache hour
+        cached_hour = datetime.datetime.now(datetime.UTC).strftime("%H")
+        cached_hits = self.cached_hits.get(cached_hour, 0)
```

Comment:
Whole code typo? Shouldn't it be 1000 per day?

Reply:
Yeah, I was convinced it was per hour, sorry...

I'll fix that soon

Reply:
`owm` title/commit typo too.